### PR TITLE
update column and cost function

### PR DESCRIPTION
**Pull request by idanovinda**
* add new columns for IOPS and throughput
* add tooltips for cost information
* change the cost/month function
* merge default and defined config

### CODE REVIEW

File: ui/operator_ui/main.py [hunk 1]
```diff
@@ -304,29 +319,34 @@ def index():
     'nat_gateways_visible': True,
     'users_visible': True,
     'databases_visible': True,
-    'resources_visible': True,
-    'postgresql_versions': ['11','12','13'],
+    'resources_visible': RESOURCES_VISIBLE,
+    'postgresql_versions': ['11','12','13', '14'],
```
Comment (FxKu) [inline ui/operator_ui/main.py:323]:
```suggestion
    'postgresql_versions': ['11','12','13','14'],
```

File: ui/operator_ui/static/styles.css [hunk 2]
```diff
@@ -64,3 +64,56 @@ label {
...
+  top: 100%;
+  left: 50%;
+  margin-left: -5px;
+  border-width: 5px;
+  border-style: solid;
+  border-color: black transparent transparent transparent;
+}
+.tooltip:hover .tooltiptext {
+  visibility: visible;
+}
```
[the remark this answers is not in the document]
Comment (FxKu) [reply ui/operator_ui/static/styles.css:119]: let's have a new line at EOF

**Comment by FxKu** [comment]
:+1:

**Comment by idanovinda** [comment]
👍

**Comment by FxKu** [comment]
:+1:

**Comment by idanovinda** [comment]
👍

**Comment by idanovinda** [comment]
👍

**Pull request merged**